disco/settings.py

```diff
@@ -8,7 +8,10 @@
 import semantic_version
 from django.utils.translation import gettext_lazy as _
 
-from arches_for_science.settings import *
+try:
+    from arches.settings import *
+except ImportError:
+    pass
```

Comment:
Unlike conditional local/docker settings, for settings we clearly depend on @aarongundel and I [discussed](https://github.com/archesproject/disco/pull/4#discussion_r1324882541) making the dependence explicit by removing the `try`.

If `RENDERERS +=` fails because `RENDERERS` isn't defined, that's pretty easy to debug, as it would not boot the dev server, but you can imagine subtler issues later on even if the dev server boots successfully.